memory_stress.py

```diff
@@ -0,0 +1,18 @@
+"""Memory-stress a long-running CodeJail-using process."""
+
+from codejail import safe_exec
+
+GOBBLE_CHUNK = int(1e7)
+
+def main():
+    gobble = []
+    for i in xrange(int(1e7)):
```

Comment:
Yes, that's a coincidence.  For a real machine, we upped GOBBLE_CHUNK to 1e8.  Also, not sure why I used int(1e7) instead of 10000000. :)